examples/3dtiles.html

```diff
@@ -73,49 +73,67 @@
                 if (object.batchTable) {
                     return object.batchTable;
-                }
-                else if (object.parent) {
-                    return findBatchTableParent(object.parent);
+                } else if (object.parent) {
+                    return findBatchTable(object.parent);
                 } else {
                     return undefined;
                 }
             }
+
             function picking(event) {
-                var htmlInfo = document.getElementById('info');
-                htmlInfo.innerHTML = ' ';
+                var htmlInfo = document.getElementById('featureInfo');
+                // Remove content already displayed
+                while (htmlInfo.firstChild) {
+                    htmlInfo.removeChild(htmlInfo.firstChild);
+                }
 
-                const intersects = view.pickObjectsAt(
+                var intersects = view.pickObjectsAt(
                     event,
                     5,
                     $3dTilesLayerRequestVolume);
 
                 for (var i = 0; i < intersects.length; i++) {
-                    var interAttributes = intersects[i].object.geometry.attributes;
-                    if (interAttributes) {
-                        if (interAttributes._BATCHID) {
-                            var face = intersects[i].face.a;
-                            var batchID = interAttributes._BATCHID.array[face];
-                            var batchTable = findBatchTableParent(intersects[i].object);
-
-                            htmlInfo.innerHTML +='<li><b> Batch id: </b>'+ batchID +'</li>';
-                            Object.keys(batchTable).map(function(objectKey) {
-                                var value = batchTable[objectKey][batchID];
-                                // if the value is a integer or not
-                                var info = Number.isInteger(value) ? value.toString() : value.toFixed(3).toString();
-                                htmlInfo.innerHTML +='<li><b>' + objectKey.toString() + ': </b>'+ info +'</li>';
-                                return true;
-                            });
-
-                            return;
-                        }
+                    // interAttributes are glTF attributes of b3dm tiles
+                    // (i.e. position, normal, batch id)
+                    var interAttributes =
+                        intersects[i].object.geometry.attributes;
+                    if (interAttributes && interAttributes._BATCHID) {
+                        // face is a Face3 object of THREE which is a triangular
+                        // face. face.a is its first vertex
+                        var vertex = intersects[i].face.a;
+                        // get batch id of the face
+                        var batchID = interAttributes._BATCHID.array[vertex];
+                        var batchTable = findBatchTable(
+                            intersects[i].object, 0);
```

Comment:
Yep you're right. It has been removed in the following commits.

Reply:
Ok fine. 
In the same PR, it would be necessary to squash, the modifications of the same code part, in the same commit. It's more simple for the reviewing.

Reply:
That's what I tried to do while still keeping a "logical" commit division based on new features added but I might have missed some details on the road like this one. Anyway, I edited the commit ;)